inference-engine/tests/functional/shared_test_classes/src/single_layer/prior_box_clustered.cpp

```diff
@@ -84,12 +88,25 @@ void PriorBoxClusteredLayerTest::SetUp() {
     attributes.offset = offset;
     attributes.variances = variances;
 
-    auto shape_of_1 = std::make_shared<ngraph::opset3::ShapeOf>(params[0]);
-    auto shape_of_2 = std::make_shared<ngraph::opset3::ShapeOf>(params[1]);
-    auto priorBoxClustered = std::make_shared<ngraph::op::PriorBoxClustered>(
-        shape_of_1,
-        shape_of_2,
-        attributes);
+    std::shared_ptr<ngraph::op::PriorBoxClustered> priorBoxClustered;
+    if(inputType == ngraph::helpers::InputLayerType::PARAMETER) {
+      auto shape_of_1 = std::make_shared<ngraph::opset3::ShapeOf>(params[0]);
+      auto shape_of_2 = std::make_shared<ngraph::opset3::ShapeOf>(params[1]);
+      priorBoxClustered = std::make_shared<ngraph::op::PriorBoxClustered>(
+          shape_of_1,
+          shape_of_2,
+          attributes);
+    } else {
+      auto inputPrc = ngraph::element::Type_t::i64;
```

Comment:
I suggest using `inPrc` here instead of fixing this at i64

Reply:
Done, as the inPrc is "UNSPECIFIED" in the test of other plugin( like "gpu"), keep i64 as default value and use inPrc when it has valid value.